src/components/form/components/__test__/select.test.js

```diff
@@ -46,4 +46,64 @@ describe('Select component', () => {
     );
     expect(component.toJSON()).toMatchSnapshot();
   });
+  it('Should be multioption and should accept number array value', () => {
+    const spy = jest.spyOn(console, 'error').mockImplementation();
+    const component = renderer.create(
+      <Select multiple value={[1,2,3]}>
+      </Select>,
+    );
+    expect(component.toJSON()).toMatchSnapshot();
+    expect(spy).not.toBeCalled();
+    spy.mockRestore();
+  });
+  it('Should be multioption and should accept string array value', () => {
+    const spy = jest.spyOn(console, 'error').mockImplementation();
+    const component = renderer.create(
+      <Select multiple value={['1','2','3']}>
+      </Select>,
+    );
+    expect(component.toJSON()).toMatchSnapshot();
+    expect(spy).not.toBeCalled();
+    spy.mockRestore();
+  });
+  it('Should be multioption and should not accept non array value', () => {
+    const spy = jest.spyOn(console, 'error').mockImplementation();
+    const component = renderer.create(
+      <Select multiple value={1}>
+      </Select>,
+    );
+    expect(component.toJSON()).toMatchSnapshot();
+    expect(spy).toBeCalled();
+    spy.mockRestore();
+  });
+  it('Should not be multioption and should accept number value', () => {
+    const spy = jest.spyOn(console, 'error').mockImplementation();
+    const component = renderer.create(
+      <Select value={1}>
+      </Select>,
+    );
+    expect(component.toJSON()).toMatchSnapshot();
+    expect(spy).not.toBeCalled();
+    spy.mockRestore();
+  });
+  it('Should not be multioption and should accept string value', () => {
+    const spy = jest.spyOn(console, 'error').mockImplementation();
+    const component = renderer.create(
+      <Select value={'1'}>
```

Comment:
stylistic issue: prefer double quote for jsx string instead of single quote inside curly braces:
```
<Select value="1">
```